### DISCUSSION TITLE
modifying script variables from another script

### DISCUSSION THREAD

**Discussion by ghost** [stride3d/stride#1389] [Q&A] [answered]
is there a way for a script to change the value of a variable in another script if that variable is public?

**Comment by manio143** [comment, marked as answer]
A script can modify a public field/property of another script. If they are on the same entity you can use `Entity.Get<OtherScriptClass>()` to obtain a reference to the other script. If they are on different entities, then you should create a field/property `public OtherScriptClass {get;set;}` and assign it via the editor by draggin over it an entity with the other script.
